blocks/api/serializer.js

```diff
@@ -111,6 +111,11 @@ export function serializeAttributes( attrs ) {
 
 export function serializeBlock( block ) {
 	const blockName = block.name;
+
+	if ( 'core/nextpage' === blockName ) {
+		return '<!--nextpage-->';
+	}
+
```

Comment:
I know it would be nice, but it's up there to not cause errors when calling `getSaveContent()` or  `getCommentAttributes()` because this block doesn't have any of that.